collections/Lat_th/MS_Lat_th_e_22.xml

```diff
@@ -51,132 +46,176 @@
+                  <textLang mainLang="la">Latin</textLang>
+                  <msItem>
+                     <locus>(upper endleaf)</locus>
+                     <note>List of contents</note>
+                  </msItem>
                   <msItem xml:id="MS_Lat_th_e_22-item1" n="1">
+                     <locus>(fols. 1r–122v)</locus>
                      <title key="work_14583">Sermon collection</title>
-                     <rubric>Sermones quadragesimales <note>(front cover)</note>
-                     </rubric>
+                     <rubric>Sermones quadragesimales</rubric>
+                     <note>(front cover)</note>
+                     <incipit><hi rend="underline"><quote>Tu autem cum ieiunas unge capud tuum</quote> Mt vi.</hi> In hoc ieiunii capite necessarium nobis est ut taliter ieiunemus</incipit>
+                     <explicit>resurrectionem de lecto et tamen pigrescit hoc facere.</explicit>
+                     <finalRubric>Finito libro sit laus et gloria cristo. Expliciunt sermones ewangeliorum completi post <gap reason="cancelled" n="50" unit="char"/> anno domini Mº.CCCº.xciiº in die Inuencionis sancte Crucis.</finalRubric>
+                     <bibl>For other copies, see Schneyer, <title>Beobachtungen zu lateinischen Sermoneshandschriften der Staatsbibliothek München</title> (1958), no. 5197.</bibl>
+                  </msItem>
+                  <msItem xml:id="MS_Lat_th_e_22-item1-item1">
+                     <locus>(fols. 123r–124v)</locus>
+                     <title key="work_14583">Sermon</title>
+                     <rubric>Sequitur sermo in die parascheue et de sancta cruce</rubric>
                      <incipit>
-                        <hi rend="italic">Tu autem cum ieiunas unge caput tuum Mt vi.</hi> In hoc ieiunii capite necessarium nobis est ut taliter ieiunemus</incipit>
-                     <explicit>exaudita est oratio tua</explicit>
-                     <colophon>Expliciunt sermones oangeliorum completi [ ] anno domini mocccoxciio in die invencienis sancte crucis <locus>(fol. 122v)</locus>
-                     </colophon>
-                     <note>Colophon followed by two further sermons:</note>
-                     <msItem xml:id="MS_Lat_th_e_22-item1-item1">
-                        <rubric>in die parascheve et de sancta cruce</rubric>
-                        <incipit>
-                           <hi rend="italic">Baiolans sibi crucem exivit</hi> … hodic mater ecclesia universis filiis suis</incipit>
-                     </msItem>
-                     <msItem xml:id="MS_Lat_th_e_22-item1-item2">
-                        <incipit>Item preter illa que dicta sunt notanda alia quattuor bona</incipit>
-                     </msItem>
-                     <textLang mainLang="la">Latin</textLang>
+                        <hi rend="underline"><quote>Baiolans sibi crucem exiuit in eum qui dicitur calvarie locum</quote> Io. 20.</hi> Hodie mater ecclesia uniuersis filiis suis</incipit>
```

Comment:
Schneyer's entry seems to treat this as the final item of the preceding collection